examples/quantization_aware_training/tensorflow/mobilenet_v2/main.py

```diff
@@ -0,0 +1,200 @@
+
+###############################################################################
+# Quantize a Tensorflow model
+#
+# The transformation function transforms a data item into model input data.
+#
+# To validate the transform function use the following code:
+# >> for data_item in val_loader:
+# >>    model(transform_fn(data_item))
+
+
+def transform_fn(data_item):
+    images, _ = data_item
+    return images
+
+
+# The calibration dataset is a small, no label, representative dataset
+# (~100-500 samples) that is used to estimate the range, i.e. (min, max) of all
+# floating point activation tensors in the model, to initialize the quantization
+# parameters.
+#
+# The easiest way to define a calibration dataset is to use a training or
+# validation dataset and a transformation function to remove labels from the data
+# item and prepare model input data. The quantize method uses a small subset
+# (default: 300 samples) of the calibration dataset.
+
+calibration_dataset = nncf.Dataset(val_dataset, transform_fn)
+tf_quantized_model = nncf.quantize(tf_model, calibration_dataset)
+
+tf_quantized_model.compile(
+    optimizer=tf.keras.optimizers.Adam(learning_rate=1e-5),
+    loss=tf.keras.losses.CategoricalCrossentropy(),
+    metrics=[tf.keras.metrics.CategoricalAccuracy()],
+)
+
+tf_quantized_model.fit(train_dataset, epochs=3, verbose=1)
+
+###############################################################################
+# Benchmark performance, calculate compression rate and validate accuracy
+
+ov_model = ov.convert_model(tf_model, share_weights=False)
+stripped_model = nncf.strip(tf_quantized_model)
+ov_quantized_model = ov.convert_model(stripped_model, share_weights=False)
+
+fp32_ir_path = ROOT / "mobilenet_v2_fp32.xml"
+ov.save_model(ov_model, fp32_ir_path, compress_to_fp16=False)
+print(f"[1/7] Save FP32 model: {fp32_ir_path}")
+fp32_model_size = get_model_size(fp32_ir_path)
+
+int8_ir_path = ROOT / "mobilenet_v2_int8.xml"
+ov.save_model(ov_quantized_model, int8_ir_path)
+print(f"[2/7] Save INT8 model: {int8_ir_path}")
+int8_model_size = get_model_size(int8_ir_path)
+
+print("[3/7] Benchmark FP32 model:")
+fp32_fps = run_benchmark(fp32_ir_path, shape=[1, 224, 224, 3])
+print("[4/7] Benchmark INT8 model:")
+int8_fps = run_benchmark(int8_ir_path, shape=[1, 224, 224, 3])
+
+print("[5/7] Validate OpenVINO FP32 model:")
+fp32_top1 = validate(ov_model, val_dataset)
+print(f"Accuracy @ top1: {fp32_top1:.3f}")
+
+print("[6/7] Validate OpenVINO INT8 model:")
+int8_top1 = validate(ov_quantized_model, val_dataset)
+print(f"Accuracy @ top1: {int8_top1:.3f}")
+
+print("[7/7] Report:")
+print(f"Accuracy drop: {fp32_top1 - int8_top1:.3f}")
+print(f"Model compression rate: {fp32_model_size / int8_model_size:.3f}")
+# https://docs.openvino.ai/latest/openvino_docs_optimization_guide_dldt_optimization_guide.html
+print(f"Performance speed up (throughput mode): {int8_fps / fp32_fps:.3f}")
```

Comment:
Similar Torch quantization aware training example outputs a table with results. Maybe we can align the examples and use the same table format here IMHO
https://github.com/openvinotoolkit/nncf/blob/develop/examples/quantization_aware_training/torch/resnet18/main.py#L338-L355

Reply:
This example is aligned with the TF PTQ example. As far as I know, there are no requirements for aligning examples across backends. If there are, let's discuss them.